students/alex_skrn/lesson02/grid.py

```diff
@@ -0,0 +1,112 @@
+"""Print a 2x2 grid, define two print_grid functions."""
+# PART 1 - print a 2x2 grid, each cell 4 dashes large
+
+plus, minus, bar, space = '+', '-', '|', ' '
+numcells = 2
```

Comment:
for two worded variables, you'll want to separate them via a `_`.  So `num_cells` and `num_dashs`.  This makes them more readable and is the standard python follows.